scalafix-core/shared/src/main/scala/scalafix/internal/config/LazySemanticdbIndex.scala

```diff
@@ -12,20 +12,20 @@ import org.langmeta.io.AbsolutePath
 // the moment we instantiate the rule.
 //type LazySemanticdbIndex = RuleKind => Option[SemanticdbIndex]
 class LazySemanticdbIndex(
-    f: RuleKind => Option[SemanticdbIndex],
-    val reporter: ScalafixReporter,
-    val workingDirectory: AbsolutePath
+    f: RuleKind => Option[SemanticdbIndex] = _ => None,
+    val reporter: ScalafixReporter = ScalafixReporter.default,
+    // The working directory when compiling file:relativepath/
```

Comment:
Oh, looks like the fix to the bug that I mentioned the other day! 👍

Reply:
You will also be happy to know sbt-scalafix `file:` tab completes absolute paths and `..` paths! #408